services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/RateDetailsSummarySectionV2.test.tsx

```diff
@@ -400,6 +401,75 @@ describe('RateDetailsSummarySection', () => {
         ).not.toBeInTheDocument()
     })
 
+    it('renders the deprecated rate programs when present on a locked historic linked rate', async () => {
+        const statePrograms = mockMNState().programs
+        const contract = mockContractPackageDraft()
+        contract.draftRates![0].draftRevision!.formData.deprecatedRateProgramIDs =
+            [statePrograms[0].id]
+        contract.draftRates![0].draftRevision!.formData.rateProgramIDs = []
+        contract.draftRates![0].draftRevision!.submitInfo = {
+            updatedAt: new Date('01/02/2024'),
+            updatedBy: 'zuko@example.com',
+            updatedReason: 'initial submission',
+        }
+        contract.draftRates![0].parentContractID = contract.id
+        contract.draftRates![0].draftRevision!.unlockInfo = null
+        await waitFor(() => {
+            renderWithProviders(
+                <RateDetailsSummarySection
+                    contract={contract}
+                    submissionName="MN-MSHO-0003"
+                    statePrograms={statePrograms}
+                    editNavigateTo="/edit"
+                />,
+                {
+                    apolloProvider: apolloProviderCMSUser,
+                }
+            )
+        })
+
+        expect(
+            screen.queryByRole('definition', {
+                name: 'Programs this rate certification covers',
+            })
+        ).toBeInTheDocument()
+
+        expect(
+            screen.queryByRole('definition', {
+                name: 'Rates this rate certification covers',
+            })
+        ).not.toBeInTheDocument()
+    })
+
+    it('does not render the deprecated rate programs when present on an unlocked historic linked rate', async () => {
+        const statePrograms = mockMNState().programs
+        const contract = mockContractPackageSubmitted()
```

Comment:
Where is there an "unlocked historic linked rate" here? I"m not seeing it in the test data but let me know what I'm missing

Reply:
Sorry that's a typo in the description